src/app/plugins/scores.plugin.ts

```diff
@@ -0,0 +1,180 @@
+
+
+        visitorData[0] = visitorData[0].slice(1); //Remove the prefix that starts the visitors name
+
+        const visitorName = this._getTeamNameFromTeamData(visitorData);
+        const homeName = this._getTeamNameFromTeamData(homeData);
+
+        const visitorScore = parseInt(visitorData[visitorData.length - 1]);
+        const homeScore = parseInt(homeData[homeData.length - 1]);
+
+        //Boolean for if the searched team is the one winning
+        let winning = 0;
+        let opponentName = '';
+        let teamName = '';
+
+        //If searched team is the visitor, else
+        if (this._containsAllTokens(visitorName, teamArg.split(' '))) {
+
+          if (visitorScore > homeScore) { winning = 1; }  //Winning
+          else if (visitorScore < homeScore) { winning = 0; } //Losing
+          else if (visitorScore == homeScore) { winning = 2; } //Tied
+
+          opponentName = homeName;
+          teamName = visitorName;
+        } else {
+
+          if (homeScore > visitorScore) { winning = 1; }  //Winning
+          else if (homeScore < visitorScore) { winning = 0; } //Losing
+          else if (homeScore == visitorScore) { winning = 2; } //Tied
+
+          opponentName = visitorName;
+          teamName = homeName;
+        }
+
+        let winningString;
+        switch (winning) {
+          case 0: winningString = 'losing';
+            break;
+          case 1: winningString = 'winning';
+            break;
+          case 2: winningString = 'tied';
+            break;
+          default:
+            winningString = 'unknown';
+            break;
+        }
+
+        const embed = new RichEmbed();
+        embed.setTitle(visitorName + ' at ' + homeName);
+        embed.setColor('#7289da');
+        embed.setDescription(
+          teamName + ' is currently ' + winningString + ' against ' + opponentName
+        );
+        embed.setFooter(visitorScore + ' - ' + homeScore);
+        embedBuffer.push(embed);
+        messagesQueued++;
+
+      }
+
+      //Send embeds in buffer
+      while (embedBuffer.length > 0) {
+        message.channel.send(embedBuffer.pop());
+      }
+
+      if (!teamFound) { message.reply('Team not found!'); }
+
+    } catch (error) {
+      this.container.loggerService.error(error);
+    }
+  }
+
+  private _getTeamNameFromTeamData(arr: string[]): string {
+    let name = '';
+    for (let i = 0; i < arr.length - 1; i++) { name += arr[i] + ' '; }
+
+    //Trim whitespace
+    while (name[0] == ' ' || name[0] == '^') { name = name.slice(1); }
+    while (name[name.length - 1] == ' ' || name[name.length - 1] == '^') {
+      name = name.slice(0, name.length - 1);
+    }
+
+    return name;
+  }
+
+  private _containsAllTokens(string: string, tokens: string[]): boolean {
+    //Return false if one of the tokens in not in string
+    for (const token of tokens) {
+      if (!string.toLowerCase().includes(token.toLowerCase())) { return false };
+    }
+
+    return true;
+  }
```

Comment:
Don't name the parameter `string` ... confusing with type `string`. 

Also, instead of for loop:
```js
sl = str.toLowerCase()
tokens.map(t => t.toLowerCase()).every(t => sl.includes(t))
```